src/lib.rs

```diff
@@ -1,197 +1,56 @@
-//! # Serve the web
-//!
-//! Tide is a friendly HTTP server built for casual Rustaceans and veterans alike. It's completely
-//! modular, and built directly for `async/await`. Whether it's a quick webhook, or an L7 load
-//! balancer, Tide will make it work.
-//!
-//! # Features
-//!
-//! - __Fast:__ Written in Rust, and built on Futures, Tide is incredibly efficient.
-//! - __Friendly:__ With thorough documentation, and a complete API, Tide helps cover your every
-//!     need.
-//! - __Minimal:__ With only a few concepts to learn, Tide is easy to pick up and become productive
-//!     with.
+//! Tide is a minimal and pragmatic Rust web application framework built for
+//! rapid development. It comes with a robust set of features that make
+//! building database-backed web applications easier and more fun.
```

Comment:
same comment here as in cargo.toml — not all tide apps are database-backed, nor does tide really do anything special to enable database-backed apps